README.md

```diff
@@ -88,13 +71,13 @@ function listenForOutsideClicks(listening, setListening, menuRef, setIsOpen) {
     if (listening) return;
     if (!menuRef.current) return;
     setListening(true);
-    [`click`, `touchstart`].forEach((type) => {
-      document.addEventListener(`click`, (evt) => {
+    [`click`, `touchstart`].forEach(type => {
```

Comment:
following standard `prettier` formatting, no exception to parentheses. None, one, or more, all use parens.

Reply:
please still restore this to its previous form.